src/js/storyTelling-story.js

```diff
@@ -59,9 +65,40 @@ https://raw.githubusercontent.com/waharnum/sjrk-storyTelling/master/LICENSE.txt
             "{story}.events.onStoryListenToRequested":
             {
                 func: "{that}.queueSpeech",
-                args: ["{that}.model.storyText"]
+                args: ["@expand:{story}.getFullStoryText()"]
             }
         }
     });
 
+    // Generates full story text, including the title, author name and story content.
+    // title: the title of the story
+    // author: the author's name/moniker
+    // content: the textual content of the story
+    // bylineTemplate: the template string for the byline,
+    //      the value "%storyAuthor" will be replaced by the author's name
+    sjrk.storyTelling.story.getFullStoryText = function (title, author, content, bylineTemplate) {
```

Comment:
Could this function be simplified by making use of `fluid.stringTemplate` in places?